### DISCUSSION TITLE
Wie funktionieren Bindings?

### DISCUSSION THREAD

**Discussion by Blockmove** [iobroker-community-adapters/ioBroker.webui#229] [General] [answered]
Ich finde den Adapter richtig interessant.
Vieles funktioniert deutlich besser als in der vis 2.
Was mich interessieren würde:
Wie funktionieren Bindings? 
Ich möchte konkrekt den Min- und Max-Wert eines numerischen Eingabefeldes über Bindings vorgeben.
Ist dies möglich?

**Comment by jogibear9988** [comment, marked as answer]
https://github.com/iobroker-community-adapters/ioBroker.webui/wiki/DE%3ABindings
